### ISSUE TITLE
Multiple inputs multiple outputs support for tensorflow serving

### ISSUE THREAD

**Issue by 3nomis** [tensorflow/tensorflow#34710]
Hi I cannot deploy my keras-TF model via tensorflow serving because I have 4 inputs. Is there any chance to do it in other ways?

**Comment by ravikyram** [comment]
This issue is more suitable for TensorFlow Serving repo. Please post it on Serving repo from [here.](https://github.com/tensorflow/serving/issues) Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
